src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -8,6 +8,6 @@
 @Repository
 public interface UserRepository extends JpaRepository<Users, Long> {
 
-    Optional<Users> findByName(String name);
+    Users findByName(String name);
```

Comment:
@MatiasVillalba96 I think that this method should return optional and in controller validate this method with lambda.